mender.go

```diff
@@ -43,7 +45,10 @@ type Controller interface {
 const (
 	defaultManifestFile = "/etc/mender/build_mender"
```

Comment:
On Fri, Sep 9, 2016 at 8:42 AM, Kristian Amlie <notifications@github.com>
wrote:

> In mender.go
> <https://github.com/mendersoftware/mender/pull/108#discussion_r78135057>:
>
> > @@ -43,7 +45,10 @@ type Controller interface {
> >  const (
> >  	defaultManifestFile = "/etc/mender/build_mender"
>
> Did you forget to update this one?
>
Yup, looks like it.